### ISSUE TITLE
Link to example of Kafka Streams usage is not working.

### ISSUE THREAD

**Issue by liweiz** [Yolean/kubernetes-kafka#103]
In section Start Kafka of README:

> That's it. Just add business value 😉. For clients we tend to use librdkafka-based drivers like node-rdkafka. To use Kafka Connect and Kafka Streams you may want to take a look at our sample Dockerfiles.

The link for Dockerfiles is not working. Based on the content of the link (github.com/solsson/dockerfiles/tree/master/streams-logfilter), I suppose it was for the stream example? I have not found there is a folder named with stream in that repo.

May I know where I should look at for the stream example?

Thanks in advance,

Liwei

**Comment by solsson** [comment]
Thanks for pointing this out. I wrote that before I had grasped the fundamental difference between Connect and Streams. See the commit comment above. For Streams I recommend the introduction videos https://www.youtube.com/watch?v=Z3JKCLG3VP4, which explains that it's just a lib. The only thing specific to kubernets-kafka once you've built your Streams-based image will be the boostrap.servers configuration string.

I'd still like to set up these examples, but unfortunately I can't prioritize that right now.
